XmlExample/meshAttributes.xml

```diff
@@ -1,8 +1,8 @@
 <freeSurface>1</freeSurface>
 <dynamicRupture>2</dynamicRupture>
 <absorbing>3,4</absorbing>
-<boundaryCondition tag="7">7</boundaryCondition>
-<boundaryCondition tag="8">8,9</boundaryCondition>
+<boundaryCondition tag="65">7</boundaryCondition>
```

Comment:
then maybe add: 
```<comment>see https://seissol.readthedocs.io/en/latest/fault-tagging.html for SeisSol convention regarding boundaryCondition</comment>```

Reply:
Why this comment format and not the standard xml-comment:

```xml
<!-- see https://seissol.readthedocs.io/en/latest/fault-tagging.html for SeisSol convention regarding boundaryCondition -->
<boundaryCondition tag="65">7</boundaryCondition> 
```
With this, you get nice syntax highlighting :)

Reply:
ha yes, that's better thanks!